packages/portal-api/src/ts/handler.ts

```diff
@@ -0,0 +1,34 @@
+import { addressUtils, logUtils } from '@0xproject/utils';
+import * as express from 'express';
+import * as _ from 'lodash';
+import { Op } from 'sequelize';
+
+import { models } from './models';
+
+export const handler = {
+    async getPrices(req: express.Request, res: express.Response) {
```

Comment:
`getPricesAsync` -> our linter should have caught this.

Reply:
hm, maybe its because handler is just an object and not a class? I'll open an issue